client/packages/plugins/backend-plugin-webpack.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = ({ distDir }) => ({
   resolve: {
     extensions: ['.js', '.ts'],
     alias: {
-      '@backendPlugins': path.resolve(__dirname, 'backendTypes/types.ts'),
+      '@common': path.resolve(__dirname, 'backendCommon'),
```

Comment:
Since utility method were added i change the naming, didn't think there would be too much diff